pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
+py36 = true
```

Comment:
this needs to be false, because the codebase also supports Python 2.7

Reply:
Removed this; but seems like default (and I'm assuming false) means `per-file auto-detection` - wouldn't that lead to mixed syntax? Some files formatted one way and others another way.